src/stream.c

```diff
@@ -837,14 +837,20 @@ int check_cc(adapter *ad)
 	int pid;
 	int packet_no_sid = 0;
 	unsigned char *b;
+        
+	if (ad->buf[0] != 0x47) {
```

Comment:
I am not sure this looks right, you are skipping the entire method for one broken packet.
I think is better if we log it and skip processing it.

Reply:
ok did remove that part